### PR TITLE
Clarify friction implementation in the NSFV action 

### PR DESCRIPTION
**Pull request by kyriv-lab**
This Pr resolves issue #27634

### CODE REVIEW

File: modules/navier_stokes/include/base/NSFVBase.h [hunk 1]
```diff
@@ -557,8 +557,9 @@ NSFVBase<BaseType>::commonMomentumEquationParams()
       "'porous_medium_treatment' is enabled, the coefficients already contain a velocity "
       "multiplier but they are not multiplied with density yet!");
 
-  params.addParam<bool>(
-      "standard_friction_formulation", true, "Flag to enable standard friction formulation");
+  params.addParam<bool>("standard_friction_formulation",
+                        true,
+                        "Flag to enable the standard friction formulation or it's alternative.");
```
Comment (GiudGiud) [inline modules/navier_stokes/include/base/NSFVBase.h:562]:
```suggestion
                        "Flag to enable the standard friction formulation or its alternative.");
```
describe what the alternative here, a simplified model (see user documentation)

File: modules/navier_stokes/include/base/NSFVBase.h [hunk 2]
```diff
@@ -3484,9 +3485,6 @@ template <class BaseType>
 void
 NSFVBase<BaseType>::checkFrictionParameterErrors()
 {
-  if (parameters().isParamSetByUser("friction_types"))
-    checkDependentParameterError("friction_types", {"standard_friction_formulation"}, true);
```
Comment (GiudGiud) [inline modules/navier_stokes/include/base/NSFVBase.h:3488]: what's up with that? do we not want standard anymore?

**Review by GiudGiud — CHANGES_REQUESTED** [review]

File: modules/navier_stokes/include/base/NSFVBase.h (hunk 2 above)
Comment (kyriv1980) [reply modules/navier_stokes/include/base/NSFVBase.h:3488]: Standard is the default. We don't want to see an error when the user doesn't explicitly defines one in the action.

Comment (GiudGiud) [reply modules/navier_stokes/include/base/NSFVBase.h:3488]: ok sounds good

File: modules/navier_stokes/include/base/NSFVBase.h [hunk 3]
```diff
@@ -558,7 +558,10 @@ NSFVBase<BaseType>::commonMomentumEquationParams()
       "multiplier but they are not multiplied with density yet!");
 
   params.addParam<bool>(
-      "standard_friction_formulation", true, "Flag to enable standard friction formulation");
+      "standard_friction_formulation",
+      true,
+      "Flag to enable the standard friction formulation or it's alternative, "
```
Comment (GiudGiud) [inline modules/navier_stokes/include/base/NSFVBase.h:563]:
```suggestion
      "Flag to enable the standard friction formulation or its alternative, "
```

**Comment by GiudGiud** [comment]
merge the two commits pls

File: python/MooseDocs/tree/html.py [hunk 4]
```diff
@@ -101,7 +101,7 @@ class String(NodeBase):
     """
     def __init__(self, parent=None, **kwargs):
         kwargs.setdefault('content', '')
-        kwargs.setdefault('escape', '')
+        kwargs.setdefault('escape', False)
```
Comment (GiudGiud) [inline python/MooseDocs/tree/html.py:104]: this should not be included

Comment (kyriv1980) [reply python/MooseDocs/tree/html.py]: I have no idea how this file was edited?

Comment (GiudGiud) [reply python/MooseDocs/tree/html.py]: it got changed by another commit which you must have squashed in yours

Comment (kyriv1980) [reply python/MooseDocs/tree/html.py]: Hmmm, anyway... fixed it.

File: modules/navier_stokes/include/base/NSFVBase.h (hunk 3 above)
Comment (GiudGiud) [reply modules/navier_stokes/include/base/NSFVBase.h:563]: this isnt fixed

Comment (kyriv1980) [reply modules/navier_stokes/include/base/NSFVBase.h]: done.

**Review by GiudGiud — APPROVED** [review]

**Pull request merged**